meta-iotqa/conf/test/refkit-image-common-extensive.manifest

```diff
@@ -1 +1,3 @@
 # Extensive tests for common profile
+oeqa.runtime.bluetooth.comm_bt_command
```

Comment:
Thanks @ipuustin, I tested it and it worked. The main issue with this approach, however, is that the objective is to execute these tests for all current profile images, so the configuration would need to be adjusted for all of them.